release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -602,6 +602,12 @@ For more information, see xref:../installing/installing_with_agent_based_install
 [id="ocp-release-notes-openshift-cli_{context}"]
 === OpenShift CLI (oc)
 
+oc-mirror plugin v2 is now generally available. To use it, add the `--v2` flag when running oc-mirror commands, see xref:../../disconnected/mirroring/oc-mirror-migration-v1-to-v2.adoc[Migration from v1 to v2 docs]. The previous version (oc-mirror plugin v1), which runs when the `--v2` flag is not set, is now deprecated. We encourage customers to transition to oc-mirror plugin v2 for continued support and improvements.
+
+oc-mirror plugin v2 now supports mirroring helm charts. Also, oc-mirror plugin v2 can now be used in environments where HTTP/S proxy is enabled, ensuring broader compatibility with enterprise setups.
+
+oc-mirror plugin v2 introduces retro-compatible filtering of Operator catalogs and generates filtered catalogs. This feature allows cluster administrators to view only the Operators that have been mirrored, rather than the complete list from the origin catalog.
+
```

Comment:
do we want to mention something about the release signature config map being generated in v2 now?

If we don't have a known issue for the previous release, it probably isn't too important?

@lmzuccarelli WDYT?

Reply:
@sherine-k - Agreed not too important